public/index.html

```diff
@@ -2,12 +2,21 @@
 <html>
   <head>
     <meta charset="utf-8">
+    <link href="https://fonts.googleapis.com/css?family=Roboto:300,400,700" rel="stylesheet">
     <link rel="stylesheet" href="./css/styles.css">
     <title>Palette Picker</title>
   </head>
   <body>
     <header>
-      <h1>Palette Picker</h1>
+      <h1>
+        <script type="text/javascript">
```

Comment:
Writing inline javascript is kind of hacky/not best practice. I would like to learn SASS or another CSS preprocessor to eliminate the need to have this.